src/Symfony/Component/Cache/Adapter/RedisTagAwareAdapter.php

```diff
@@ -161,46 +148,82 @@ protected function doDelete(array $ids, array $tagData = []): bool
      */
     protected function doInvalidate(array $tagIds): bool
     {
-        if (!$this->redisServerSupportSPOP()) {
+        if (($redis = $this->redis) instanceof \Predis\ClientInterface && $redis->getOptions()->exceptions) {
```

Comment:
Should we rather forbid this case entirely rather than creating a new Redis connection for each invalidation (and relying on altering a private property, which excludes us from *any* BC guarantee of the lib) ?

Reply:
I moved this part to the constructor of the adapter to it's done only once.